sdk/communication/Azure.Communication.Common/api/Azure.Communication.Common.netstandard2.0.cs

```diff
@@ -53,6 +54,14 @@ public CommunicationUserIdentifier(string id) { }
         public override bool Equals(Azure.Communication.CommunicationIdentifier other) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class EntraCommunicationTokenCredentialOptions
+    {
+        public EntraCommunicationTokenCredentialOptions(string resourceEndpoint, Azure.Core.TokenCredential entraTokenCredential) { }
+        public EntraCommunicationTokenCredentialOptions(string resourceEndpoint, Azure.Core.TokenCredential entraTokenCredential, string[] scopes) { }
```

Comment:
Instead of adding a second constructor with scopes, we should add a setter to the Scopes property. Sorry for not catching that earlier.

Reply:
That's mainly to avoid having too many constructors when more properties get added over time and are optional.

Reply:
fixed, fetched the latest changes from the main branch and merged :)